packages/asc-ui/src/components/Select/Select.tsx

```diff
@@ -6,20 +6,28 @@ const Select: React.FC<Props & React.HTMLAttributes<HTMLSelectElement>> = ({
   id,
   value,
   label,
-  error,
+  error: errorProp,
+  errorMessage,
   ...otherProps
-}) => (
-  <>
-    {label && <FormLabelStyle htmlFor={id}>{label}</FormLabelStyle>}
-    <SelectStyle
-      {...{
-        ...otherProps,
-        id,
-        value,
-        error,
-      }}
-    />
-  </>
-)
+}) => {
+  const error = errorProp || errorMessage
+
+  return (
+    <>
+      {label && <FormLabelStyle htmlFor={id}>{label}</FormLabelStyle>}
+      {errorMessage && (
+        <FormLabelStyle htmlFor={id}>{errorMessage}</FormLabelStyle>
```

Comment:
doesnt this need to be red? (passing the an "error" prop or sth?)

Reply:
Correct! I was waiting for the other PR to close so I could implement (-: Pushing now